src/lib/bootloader/grub2efi.rb

```diff
@@ -59,7 +59,7 @@ def propose
       # for UEFI always remove PMBR flag on disk (bnc#872054)
       self.pmbr_action = :remove
 
-      grub_default.generic_set("GRUB_USE_LINUXEFI", Yast::Arch.aarch64 ? "false" : "true")
+      grub_default.generic_set("GRUB_USE_LINUXEFI", (Yast::Arch.aarch64 || Yast::Arch.arm) ? "false" : "true")
```

Comment:
IMO should be left as `false` everywhere.
If that's too risky, it should check for x86 specifically though as only that has `linuxefi`.

Reply:
So something like:
```
grub_default.generic_set("GRUB_USE_LINUXEFI", (Yast::Arch.x86_64 || Yast::Arch.i386) ? "true" : "false")
```
?

Reply:
Yep

Reply:
Done